capstone/src/main/webapp/script.js

```diff
@@ -12,11 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+window.addEventListener('DOMContentLoaded', (event) => {
+  if (document.getElementById('map')) {
+    createMap();
+  }
+  if (document.getElementById('filters-form')) {
+    getInputFilters();
+  }
+});
+
+var map;
+var center = new google.maps.LatLng(40.7128, -74.0060);
```

Comment:
for magic numbers like this I think it is best practice to include comments indicating what it represents.